azuredevops/internal/service/git/resource_git_repository.go

```diff
@@ -227,14 +227,18 @@ func createGitRepository(clients *client.AggregatedClient, repoName *string, pro
 	return createdRepository, nil
 }
 
-func initializeGitRepository(clients *client.AggregatedClient, repo *git.GitRepository) error {
+func initializeGitRepository(clients *client.AggregatedClient, repo *git.GitRepository, defaultBranch *string) error {
```

Comment:
Could you also create a test for this change?

Reply:
sure thing! have added a test.